runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -78,9 +82,11 @@
 {
 	private static final int TRAWLER_SHIP_REGION_NORMAL = 7499;
 	private static final int TRAWLER_SHIP_REGION_SINKING = 8011;
-
+	private static final int TRAWLER_TIME_LIMIT = 614;
```

Comment:
What is this in? seconds?

Reply:
Yes, current trawler time says 12 mins. But it is actually 614 seconds